### PR TITLE
package.json: split tests

### PR DESCRIPTION
**Pull request by XhmikosR**
Now it's possible to run mocha individually too :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,8 @@
   },
   "scripts": {
     "lint": "standard && node ./lib/rtlcss.js",
```
Comment (XhmikosR) [inline package.json:43]: ideally this should be split too so one can do `npm run lint -- --fix`

Comment (MohammadYounes) [reply package.json:43]: Would you update the PR ? and Thanks for contributing!

**Comment by XhmikosR** [comment]
Oops, forgot I had this opened and deleted my fork. Let me make a new PR.

File: package.json (hunk 1 above)
Comment (XhmikosR) [reply package.json:43]: How do you want to name the new `node ./lib/rtlcss.js` script? Also, do you want me to add a `lint:fix` script?

Comment (MohammadYounes) [reply package.json:43]: I was thinking 😅 

```json
"lint": "standard && node ./lib/rtlcss.js",
"lint:fix": "standard --fix && node ./lib/rtlcss.js",
```

Comment (XhmikosR) [reply package.json:43]: The thing with this approach is that doing `npm run lint -- --fix` won't work as expected. So, ideally, the `node ./lib/rtlcss.js` should be moved to a separate one. But it's your call. Personally I sort of expect this behavior.

Comment (MohammadYounes) [reply package.json:43]: Both are fine with me as long as I don't need to come up with the name 😇

**Pull request closed without merging**